app_runner.rb

```diff
@@ -0,0 +1,59 @@
+require_relative 'app'
+
+class AppRunner
+  def initialize
+    @app = App.new
+  end
+
+  def run
+    puts 'Welcome to Library App!'
+
+    loop do
+      display_options
+      choice = gets.chomp.to_i
+      break if choice == 7
+
+      handle_choice(choice)
+    end
+
+    puts 'Exiting...'
+  end
+
+  private
+
+  def display_options
+    puts 'Please choose an option:'
+    puts '1. List all books.'
+    puts '2. List all people.'
+    puts '3. Create a person (teacher or student, not a plain Person).'
```

Comment:
- [ ] Please make sure that you fix these errors below when creating a teacher or a student :point_down: 
  
  ![image](https://github.com/theresetuyi/OOP-school-library/assets/73854999/56326a00-5976-4896-8074-e34cfbc53279)
  
  
  ![image](https://github.com/theresetuyi/OOP-school-library/assets/73854999/f0b281fb-b8ad-42eb-9e9e-62481d119a4d)